apps/extension/src/senders/validate.ts

```diff
@@ -34,7 +34,13 @@ export const assertValidSender = (sender?: chrome.runtime.MessageSender) => {
   if (parsedOrigin.origin !== sender.origin) {
     throw new Error('Sender origin is invalid');
   }
-  if (!(parsedOrigin.protocol in ValidProtocol)) {
+
+  if (
+    !(
+      parsedOrigin.protocol in ValidProtocol ||
+      (globalThis.__DEV__ && isHttpLocalhost(parsedOrigin))
```

Comment:
Stumbled upon this today. I was trying to connect a production Prax to the dev Dex explorer and it would silently fail. Think we should drop the requirement that the dev version of Prax is needed to connect to localhost sites:

```diff
parsedOrigin.protocol in ValidProtocol ||
- (globalThis.__DEV__ && isHttpLocalhost(parsedOrigin))
+ isHttpLocalhost(parsedOrigin)
```

@VanishMax can you open a follow PR on this?

Reply:
@grod220 no problem! https://github.com/prax-wallet/prax/pull/208